src/Grid/GridToolbar.js

```diff
@@ -89,7 +89,7 @@ class GridToolbar extends React.Component {
         }
         {
           isPrintEnabled && 
-          <IconButton onClick={onPrint}>
+          <IconButton onClick={onPrint} disabled={filteredRecordCount === 0}>
```

Comment:
The entire PR should have included only this, that's how you deal with: "Fix error when there are no records to print"